Description: Fix incorrect option handling int KateDK
 This patch make sure -V do not trigger "unknown option" warning
 and get both -V and -h to print what they should.
Author: Petter Reinholdtsen <pere@hungry.com>
Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2016-02-16

diff --git a/tools/KateDJ/KateDJ b/tools/KateDJ/KateDJ
index ae9c7ed..4f2b2cc 100755
--- a/tools/KateDJ/KateDJ
+++ b/tools/KateDJ/KateDJ
@@ -8,15 +8,15 @@ stop=False
 if len(sys.argv)>1:
   arg=sys.argv[1]
   if arg=='-V' or arg=='--version':
-    print kdj_name_version+', a remuxing program for Kate streams.\n'
+    print constants.kdj_name_version+', a remuxing program for Kate streams.\n'
     stop=True
-  if arg=='-h' or arg=='--help':
-    print kdj_name_version+', a remuxing program for Kate streams.\n'
+  elif arg=='-h' or arg=='--help':
+    print constants.kdj_name_version+', a remuxing program for Kate streams.\n'
     print '\n'+\
-          kdj_name+' is a remuxing program that allows extracting and decoding Kate tracks\n'+\
+          constants.kdj_name+' is a remuxing program that allows extracting and decoding Kate tracks\n'+\
           'from an Ogg stream, and recreating that Ogg stream after the Kate streams\n'+\
           'have been altered.\n'+\
-          kdj_name+' requires both the Kate tools (kateenc and katedec) and the oggz tools.\n'+\
+          constants.kdj_name+' requires both the Kate tools (kateenc and katedec) and the oggz tools.\n'+\
           '\n'+\
           'This is a GUI program. Click the \'Help\' button in the main window for more help.\n'
     stop=True
